src/components/autocomplete/demoRepeatMode/index.html

```diff
@@ -0,0 +1,97 @@
+<div layout="column" ng-cloak>
+  <md-content class="md-padding">
+    <div layout="row" layout-xs="column">
+      <div class="instance-wrapper" flex-gt-xs="50" ng-controller="DemoCtrl as ctrl">
+        <form ng-submit="$event.preventDefault()">
+          <p id="standard-autocomplete-description">
+            Specify the <code>md-mode</code> of <code>standard</code> to fall back to
+            <code>ng-repeat</code> for autocomplete entries. This allows you to have
+            list elements of varying heights. Note that sizable lists can have
+            performance implications for your application. Using <code>standard</code>
+            mode will require you to address those directly within your application.
```

Comment:
A slight refinement here:
```
             Specify the <code>md-mode</code> as <code>standard</code> to fall back to
             <code>ng-repeat</code> for the autocomplete options. This allows
             options in the list to have varying heights. Note that large lists in
             <code>standard</code> mode will have a negative performance impact
             on your application.
```
I removed the last sentence because I don't think that we should ask them to address the issue in their application w/o giving them some more specific guidance.